src/qt/walletframe.cpp

```diff
@@ -25,9 +30,25 @@ WalletFrame::WalletFrame(const PlatformStyle *_platformStyle, BitcoinGUI *_gui)
     walletFrameLayout->setContentsMargins(0,0,0,0);
     walletFrameLayout->addWidget(walletStack);
 
-    QLabel *noWallet = new QLabel(tr("No wallet has been loaded."));
+    // hbox for no wallet
+    QGroupBox* no_wallet_group = new QGroupBox(walletStack);
+    QVBoxLayout* no_wallet_layout = new QVBoxLayout(no_wallet_group);
+
+    QLabel *noWallet = new QLabel(tr("No wallet has been loaded.\nGo to File > Open Wallet to load a wallet.\n- OR -"));
```

Comment:
wouldn't it be nicer to just make a `open_wallet_button` here like it is done below for the create button? That would save the user a bit of mousing around. Also, it would remove the headache of making sure that `File > Open Wallet` stays consistent with how the menu items are actually called

Reply:
It would, but the way that opening wallets works now would require more extensive changes with code that I'm not too familiar with.

Reply:
if this were Javascript, I would simply make a button that clicks the menu item. But I guess that won't fly in Qt :-)

Reply:
The menu item doesn't open a selection dialog, it opens a submenu listing all of the wallets. That doesn't really work here.

Reply:
Do you think we should replace the submenu with a dialog?

Reply:
I suppose this could just be a dropdown listing all of the available wallets. But, as I said, I leave this up to someone more familiar with the GUI to do.

Reply:
if it were up to me I'd say a simple filepicker would totally suffice. It's self-explanatory and flexible, and the more fancy stuff is still available from the menu for those who want it

Reply:
We would need #15204 to do a file picker type thing.

Reply:
ah, sorry, I somehow thought that was already merged

Reply:
@flack see https://github.com/bitcoin/bitcoin/pull/15204#issuecomment-539932726.